src/tests/TestUtils.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
```

Comment:
`TestUtils` is a very general name, I think it has risk of unrelated test helpers getting bundled here. Do you think these helpers are useful for non-`Stream` related tests, or could it be something like `StreamTestUtils`?